backend/package.json

```diff
@@ -8,41 +8,41 @@
     },
     "private": true,
     "dependencies": {
-        "@bugsnag/js": "^7.9.0",
-        "@bugsnag/plugin-express": "^7.9.0",
+        "@bugsnag/js": "^7.11.0",
+        "@bugsnag/plugin-express": "^7.11.0",
         "axios": "^0.21.1",
         "body-parser": "^1.19.0",
         "connect-mongo": "^3.2.0",
         "cors": "^2.8.5",
         "express": "^4.17.1",
-        "express-session": "^1.17.1",
+        "express-session": "^1.17.2",
         "lodash": "^4.17.20",
         "lodash-es": "^4.17.15",
         "moment": "^2.29.1",
-        "mongoose": "^5.12.3",
-        "mongoose-aggregate-paginate": "^2.0.1",
-        "mongoose-aggregate-paginate-v2": "^1.0.42",
         "mongoose-paginate": "^5.0.3",
         "morgan": "^1.10.0",
         "passport": "^0.4.1",
         "passport-discord": "^0.1.4",
-        "redis": "^3.1.1",
+        "redis": "^3.1.2",
         "request": "^2.88.2",
         "request-promise-native": "^1.0.9",
         "swagger-jsdoc": "^6.1.0",
         "swagger-ui-express": "^4.1.4",
         "un-eval": "^1.2.0"
     },
     "devDependencies": {
-        "@types/mongoose-paginate": "^5.0.8",
-        "@types/node": "^12.11.1",
-        "@types/swagger-schema-official": "^2.0.21",
+        "@types/mongoose-paginate": "^5.0.10",
+        "@types/node": "^12.20.20",
+        "@types/swagger-schema-official": "^2.0.22",
         "cross-env": "^7.0.2",
-        "eslint": "^7.25.0",
+        "eslint": "^7.32.0",
         "eslint-config-prettier": "^8.3.0",
-        "eslint-plugin-prettier": "^3.4.0",
-        "nodemon": "^2.0.4",
-        "pm2": "^4.5.6",
-        "prettier": "^2.2.1"
+        "eslint-plugin-prettier": "^3.4.1",
+        "mongoose": "^5.13.7",
```

Comment:
Why did you move mongoose and the aggregate-paginate libs to dev dependency?

Reply:
it was because I was using --savedev and that puts them there. Moved back in 
ced7639